src/components/SecondaryNav/SecondaryNav.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import className from 'classnames';
+/**
+ * A navigation item to be used in the secondary nav bar
+ *
+ */
+export const NavLink = ({ active, children }) => {
```

Comment:
let's split `NavLink` out into it's own file, if this particular NavLink component is dependent on SecondaryNav maybe call it `SecondaryNavLink` so there's no naming collisions in views that use react-router's navLink component.  

This way storybook will pick it up and add it's propsTable in the addon-info docs
<img width="723" alt="Screen Shot 2019-04-08 at 3 18 37 PM" src="https://user-images.githubusercontent.com/1334131/55750602-a45ccd80-5a11-11e9-8829-a975f6b3d5c0.png">

We have to do this because of an annoying contraint with react-docgen where it can only see one component per file